### DISCUSSION TITLE
Quire Documentation Suggestion

### DISCUSSION THREAD

**Discussion by kristhayer11** [thegetty/quire#130] [Feedback]
I couldn't find anywhere in the documentation that addresses how to indicate folder structure (or folder paths) for image files. If a user creates their own subfolders (e.g. static > img > portraits), how and where do you point quire to those subfolders (either in yml OR the shortcode)?

**Comment by swambold1** [comment]
Would you not do it in the figures.yml file when you provide the path for the source? For example:

```
figure_list:
  - id: "America_L1"
    src: figures/America_L1.jpg
```

So in the example you provide above where the portraits folder is nested in the img folder, you'd make the source path:

`    src: portraits/America_L1.jpg
`
?

**Reply by kristhayer11** [reply]
Yes, that was my thought as well. However, it did not work. If that is the correct method, I may have to quit/restart/reboot to clear memory. I will give this a try. Thank you @swambold1 !
